api/controllers/group.controller.ts

```diff
@@ -0,0 +1,60 @@
+import { Request, Response } from 'express';
+import GroupService from '../secvices/group.service';
+
+class GroupController {
+    public async get(req: Request, res: Response): Promise<void> {
+        const { id } = req.params;
+        const groupService = new GroupService();
```

Comment:
Maybe it's better to make `groupService` a private property in controller? And to instantiate it in constructor.

Reply:
Done